src/DependencyInjection/NetzmachtContaoLeafletMetaModelsExtension.php

```diff
@@ -29,13 +30,13 @@ final class NetzmachtContaoLeafletMetaModelsExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container): void
     {
-        $loader = new XmlFileLoader(
+        $xmlLoader = new XmlFileLoader(
```

Comment:
@cliffparnitzky is it absolutely necessary to rename the variable?

Reply:
No, was part of development when there was another loader. I've changed it.